<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YoutubeWindow</class>
 <widget class="QDialog" name="YoutubeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YouTube Search</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item row="5" column="2">
    <widget class="QDialogButtonBox" name="buttonClose">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="buttonAdd">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelSearch">
       <property name="text">
        <string>Search:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="textSearch"/>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSearch">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QLabel" name="labelResults">
     <property name="text">
      <string>Results:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QListWidget" name="listResults">
     <property name="lineWidth">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="labelState">
     <property name="text">
      <string>&lt;state&gt;</string>
     </property>
     <property name="indent">
      <number>-13</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>textSearch</tabstop>
  <tabstop>buttonSearch</tabstop>
  <tabstop>listResults</tabstop>
  <tabstop>buttonAdd</tabstop>
  <tabstop>buttonClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
